src/data/markdown/translated-guides/en/04 Results output/300 Grafana dashboards.md

```diff
@@ -12,15 +12,21 @@ Having test results in a dashboard brings various benefits:
 
 ## Options
 
-With [Grafana](https://grafana.com/grafana/), you can create a custom dashboard to query and **visualize data from multiple sources and any type of backend**. With k6, you can stream your test results to any backend using a [custom output extension](/extensions/get-started/create/output-extensions/) or an existing [real-time output](/results-output/real-time/#service).
+With [Grafana](https://grafana.com/grafana/), you can create a custom dashboard to query and **visualize data from multiple sources and any type of backend**. 
+Using k6, you can [stream your test results to any backend](/results-output/real-time).
```

Comment:
```suggestion
Using k6, you can [stream your local test results to any backend](/results-output/real-time).
```

Based on the update on the first file here, is this only for streaming local test results?